lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -124,7 +124,7 @@
   gpgcheck:
     required: false
     choices: ['yes', 'no']
-    default: 'no'
+    default: null
     description:
       - Tells yum whether or not it should perform a GPG signature check on
```

Comment:
Could you please explicitly state that it falls back to `yum`'s default in case of `None`? This was ambiguous when I read the doc, thus I asked Martin to debug it, which resulted in that PR.

Reply:
:+1:

Reply:
@jtyr ^^^